src/main/java/com/microsoft/graph/content/MSBatchRequestContent.java

```diff
@@ -163,14 +164,18 @@ private String getHeaderValuesAsString(final List<String> list) {
 	}
 
 	private JsonObject requestBodyToJSONObject(final Request request) throws IOException, JsonParseException {
-		if (request == null || request.body() == null)
-			return null;
-		final Request copy = request.newBuilder().build();
-		final Buffer buffer = new Buffer();
-		copy.body().writeTo(buffer);
-		final String requestBody = buffer.readUtf8();
-		final JsonObject JsonObject = JsonParser.parseString(requestBody).getAsJsonObject();
-		return JsonObject;
+		if (request != null && request.body() != null) {
```

Comment:
style: Please consider not replacing the existing style, which eliminates the error conditions with early returns at the beginning of a function or a loop.

The alternative creates the arrow anti-pattern with too many indentation levels that becomes hard to track. https://blog.codinghorror.com/flattening-arrow-code/

Reply:
fixed, thanks!
I'm used to optimizing for positive conditions first and then putting the defensive statements at the beginning when possible. I'll try to remember doing it the other way around.